### ISSUE TITLE
Sentry doesn't seem to be picking up all exceptions

### ISSUE THREAD

**Issue by stephenmoloney** [mirumee/saleor#3397]
### What I'm trying to achieve

Have sentry pick up error messages and exceptions - same as those that would otherwise be emailed to the admin when sentry is not being used.

### Steps to reproduce the problem

Start saleor and have it generate errors 

```
python -c """
> 10 + '10'
> """
Traceback (most recent call last):
  File "<string>", line 2, in <module>
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```

The error does not appear in sentry.

However, if the raven test is run, the error is picked up in sentry
```
python manage.py raven test
```

### What I expected to happen

For the `TypeError: unsupported operand type(s) for +: 'int' and 'str'` to be logged by sentry

**Comment by stephenmoloney** [comment]
@jxltom 
I'm not great with `python/django`, can you give me some simple action I can take to raise an exception to test this?

**Comment by jxltom** [comment]
I mean add ```raise Exception('anyexception')``` in any of saleor's view function and accessing it from the browser, then sentry will collect the exceptions.

**Comment by NyanKiyoshi** [comment]
I believe it was fixed on #4607.
